services/director-v2/tests/unit/with_dbs/test_modules_dynamic_sidecar_docker_service_specs.py

```diff
@@ -258,7 +272,7 @@ def expected_dynamic_sidecar_spec(
                     "STORAGE_HOST": "storage",
                     "STORAGE_PORT": "8080",
                 },
-                "CapabilityAdd": ["CAP_LINUX_IMMUTABLE"],
+                "CapabilityAdd": None,
```

Comment:
is this change correct?

Reply:
Yes 100%, it is unused. I forgot to remove it from here long ago.